app/views/desfake/itineraries/index.html.erb

```diff
@@ -3,7 +3,7 @@
     <section class="section">
       <div class="row small-up-1 medium-up-2 large-up-3 card-grid">
         <% @itineraries.each do |itinerary| %>
-          <div class="column" id="itinerary_<%= itinerary.id %>">
+          <div class="column" id="itinerary_<%= itinerary.id %>" data-position="<%= itinerary.position %>" data-subscription="<%= itinerary.subscription %>">
```

Comment:
What's the reason for adding the `data-subscription` in the div?

Reply:
There's a request from the client to display the itinierary cards with different styles depending on the subscription type (free or premium) I thought it was cleaner to add it as data attributes instead of CSS classes.